tests/utils/shippable/shippable.sh

```diff
@@ -26,11 +33,42 @@ fi
 command -v python
 python -V
 
+function retry
```

Comment:
`retry` isn't currently needed, though I guess having this here makes it easier to keep the files in sync across repos.